libs/application/templates/data-protection-complaint/src/forms/ComplaintForm.ts

```diff
@@ -176,12 +176,13 @@ export const ComplaintForm: Form = buildForm({
                   {
                     links: [
                       {
-                        title: 'Póst- og fjarskiptastofnun',
-                        url: 'https://www.pfs.is/',
+                        title: 'Fjarskiptastofa',
+                        url: 'https://fjarskiptastofa.is/',
```

Comment:
Might benefit from being defined in contentful but not a big issue

Reply:
I agree, thought there was a reason for this, but I can change it.